project/searching.py

```diff
@@ -2,8 +2,11 @@
 def linear_search(arr, target):
   
   # TO-DO: add missing code
+  for item in arr:
+    if item == target:
+      return item
 
-   return -1   # not found
+  return -1   # not found
 
```

Comment:
One thing I would like to suggest is put comments next to your code for deeper understanding , other than that great job on linear search!